src/Steppenwolf/Startup.cs

```diff
@@ -35,9 +36,11 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddAutoMapper(typeof(Startup));
             services.AddRazorPages();
-            services.AddDbContext<PostgresDbContext>(options => options.UseNpgsql(
-                this.Configuration.GetConnectionString("PostgreSQL")
-            ));
+            services.AddDbContext<PostgresDbContext>(
+                options => options.UseNpgsql(
+                    this.Configuration.GetConnectionString("PostgreSQL")
+                ), 
+                ServiceLifetime.Transient);
```

Comment:
How does the tracking work with ServiceLifetime.Transient ?

Reply:
From what I tested it worked fine, the ServiceLifetime.Transient was required since we can make multiple context requests per thread(header categories, page posts, etc) and having it by default was breaking the loading of the pages. Since certain data was accessed when context was disposed.